autogpts/autogpt/.env.template

```diff
@@ -235,3 +235,12 @@ OPENAI_API_KEY=your-openai-api-key
 # AP_SERVER_PORT=8000
 # # AP_SERVER_DB_URL - Specifies what connection url the agent protocol database will connect to (Default: Internal SQLite)
 # AP_SERVER_DB_URL=sqlite:///data/ap_server.db
+
+
+################################################################################
+### White/Black Listing URLS
+################################################################################
+##WEB_POLICY - Specifies whether the url list is black list(0) or a whilte list(1)
+#WEB_POLICY=0
```

Comment:
* Formatting diverges from the rest of the file
* Rather make this an enum with values `blacklist` and `whitelist`